dj.Top restriction (#1084)
datajoint/datajoint-python: docs/src/query/operators.md

<!-- ## Join appears here in the general docs -->

## Top

> Similar to the univeral set operator, the top operator uses `dj.Top` notation. It is used to 
> restrict a query by the given `limit`, `order_by`, and `offset` parameters:

Reviewer: ```suggestion
Similar to the universal set operator, the top operator uses `dj.Top` notation. It is used to 
restrict a query by the given `limit`, `order_by`, and `offset` parameters.
```

> 

Reviewer: ```
| Parameter | Keyword Arguments | Default Argument | Description |
| -- | -- | -- | -- |
| `limit` | | 1 | Restrict the number of rows returned. |
| `order_by` | "KEY", "DESC", "ASC" | "KEY" |  |
| `offset` | | 0  |  |
```

Reviewer: Please fill out the rest of the table.  And perhaps the headings I am using (`Keyword Arguments`, `Default Argument`) are not the clearest, so feel free to replace.

> ```python

Reviewer: ```suggestion
Examples

```python
```

Session & dj.Top(limit=10, order_by='session_date')
```

The result of this expression returns the first 10 rows of `Session` and sorts them 
by their `session_date` in ascending order. 

> ### `order_by`

Reviewer: ```suggestion
```

| Example                                   | Description                                                                     |
|-------------------------------------------|---------------------------------------------------------------------------------|
| `order_by="session_date DESC"`            | Sort by `session_date` in *descending* order                                    |
| `order_by="KEY"`                          | Sort by the primary key                                                         |
| `order_by="KEY DESC"`                     | Sort by the primary key in *descending* order                                     |
| `order_by=["subject_id", "session_date"]` | Sort by `subject_id`, then sort matching `subject_id`s by their `session_date`  |

> The default values for `dj.Top` parameters are `limit=1`, `order_by="KEY"`, and `offset=0`.
> 

Reviewer: ```suggestion
```

```

The result of this expression returns the first 10 rows of `Session` and sorts them 
by their `session_date` in ascending order. 

### `order_by`

| Example                                   | Description                                                                       |
|-------------------------------------------|-----------------------------------------------------------------------------------|
| `order_by="session_date DESC"`            | Sorts by `session_date` in *descending* order                                     |

> | `order_by="KEY"`                          | Sorts by the primary key(s)                                                       |

Reviewer 2: ```suggestion
| `order_by="KEY"`                          | sort by the primary key |
| `order_by="KEY desc"                  | sort by primary key in descending order  |
```

| `order_by=["subject_id", "session_date"]` | Sorts by `subject_id`, then sorts matching `subject_id`s by their `session_date`  |

> The default values for `dj.top` parameters are `limit=1`, `order_by="KEY"`, and `offset=0`.

Reviewer 2: ```suggestion
The default values for `dj.Top` parameters are `limit=1`, `order_by="KEY"`, and `offset=0`.
```
